docs/core/tutorials/vb-with-visual-studio.md

```diff
@@ -14,7 +14,7 @@ This topic provides a step-by-step introduction to building, debugging, and publ
 
 ## Prerequisites
 
-[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.0.
+[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 1.1 or any higher version.
```

Comment:
,NET Core 1.0 and 1.1 are no longer supported, even though they can be used. And actually, 2.0 isn't supported either. I would change this to 2.1 which still has 1 or 2 years of support left.

```suggestion
[Visual Studio 2017](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs) with the ".NET Core cross-platform development" workload installed. You can develop your app with .NET Core 2.1 or any higher version.
```

Reply:
I'd build on top of @Thraka's suggestion and say "or later versions", instead of any higher version. That's our standard boilerplate.

Reply:
Ahh yes good point @mairaw I've updated my suggestion.

Reply:
Thanks @Thraka and @mairaw.. hope this helped.